### ISSUE TITLE
Documentation, General: Options

### ISSUE THREAD

**Issue by rozzilla** [madebyshape/svg-convert#13]
Another thing, IMHO, is that is difficult to understand how to use the attributes of the svgConvert's function, such as svgCleanupAttr, imgIncludeAttr, etc.
It'd be useful to explain clearer how to use these attributes.

**Comment by bymayo** [comment]
I'll update the README with a bit more detail on the options available 😀

**Comment by rozzilla** [comment]
Yup, I know and I wanted to remember it 'cause I think it's an important point 😉 have a nice day
